comp/core/secrets/secretsimpl/secrets.go

```diff
@@ -132,6 +139,27 @@ func (r *secretResolver) fillFlare(fb flaretypes.FlareBuilder) error {
 	return nil
 }
 
+func (r *secretResolver) writeDebugInfo(w http.ResponseWriter, _ *http.Request) {
+	r.GetDebugInfo(w)
+}
+
+func (r *secretResolver) handleRefresh(w http.ResponseWriter, _ *http.Request) {
+	result, err := r.Refresh()
+	if err != nil {
+		setJSONError(w, err, 500)
+		return
+	}
+	w.Write([]byte(result))
+}
+
+// setJSONError writes a server error as JSON with the correct http error code
+// NOTE: this is copied from comp/api/api/utils to avoid requiring that to be a go module
+func setJSONError(w http.ResponseWriter, err error, errorCode int) {
+	w.Header().Set("Content-Type", "application/json")
+	body, _ := json.Marshal(map[string]string{"error": err.Error()})
+	http.Error(w, string(body), errorCode)
+}
```

Comment:
I did not think it worth making the `comp/api/api/utils` package a go module so I copied this, are there thoughts from others about making the utils a go module instead of this approach?